client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MlClientDocumentationIT.java

```diff
@@ -588,14 +588,13 @@ public void testUpdateJob() throws Exception {
                 .setDescription("My description") // <2>
                 .setAnalysisLimits(new AnalysisLimits(1000L, null)) // <3>
                 .setBackgroundPersistInterval(TimeValue.timeValueHours(3)) // <4>
-                .setCategorizationFilters(Arrays.asList("categorization-filter")) // <5>
```

Comment:
Looking at `JobUpdate.java`, this is still valid. Why is it removed here in the test?

Reply:
It doesn't work any more, because you cannot configure both categorization filters and a categorization analyzer, and now every newly created job has a categorization analyzer.

You cannot update that categorization analyzer because that could completely change the way categorization is done, and ruin the stability of the categories.

Arguably we should never have let people update the categorization filters for the same reason.  It's like we don't allow people to change the detectors after a job was created - if they change something so fundamental to what the job is doing then it's not really the same job after the update.

Reply:
Cool, then it seems later PR should deprecate setting them and then we can remove it in 8.